test/integration/integration_test_fixtures.py

```diff
@@ -81,7 +81,7 @@ def __init__(self, request, server_config, backend_count=1):
     self.request = request
     self.ip_version = request.param
     assert self.ip_version != IpVersion.UNKNOWN
-    self.server_ip = "::" if self.ip_version == IpVersion.IPV6 else "0.0.0.0"
+    self.server_ip = "::1" if self.ip_version == IpVersion.IPV6 else "127.0.0.1"
```

Comment:
The benchmarking framework consumes this too. Would it be possible and safer to somehow localize this change to a derived class from `IntegrationTestBase`, specifically targeting tests for h/3, and use that?

Reply:
Good point, done in 0f9949096e739c46a05af41e6a0d881cabb29b4f.